hack/test-cmd.sh

```diff
@@ -337,12 +330,19 @@ for test in "${tests[@]}"; do
   os::cmd::try_until_text "oc get projects -o name" "project/${namespace}"
   os::test::junit::declare_suite_end
 
-  ${test}
+  if ! ${test}; then
```

Comment:
On second thought, `${test}` is a script which will run in a separate shell process, right? So that shell process will still exit and invoke its EXIT trap handler.